File: Management/AdminApi/AdminApi/Models/DeviceAlert.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Microsoft.Azure.Amqp.Framing;
using Newtonsoft.Json;

namespace AdminApi.Models
{
    public class DeviceAlert
    {
        [JsonProperty(PropertyName = "id")]
        public string id { get; set; }
        public string deviceid { get; set; }
        public string severity { get; set; }
        public bool? acknowledged { get; set; }
        public int? deviceMessagecount { get; set; }
        public double? allDevicesAverage { get; set; }
        public string reason { get; set; }
        public int? windowunit { get; set; }
        public string windowsize { get; set; }

        public DateTime? time { get; set; }
        public DateTime? acknowledgedTime { get; set; }
        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}

File: Management/AdminUI_Simple/src/app/model/DeviceAlert.ts

export class DeviceAlert {
    constructor(
        public id: string,
        public deviceid: string,
        public windowunit: number,
        public windowsize: string,
        public severity: string,
        public acknowledged: boolean,
        public reason: string,
        public allDevicesAverage: number,
        public allDevicesMedian: number,
        public deviceMessagecount: number,
        public time:Date,
        public acknowledgedTime:Date,
         ) {
    }
}
